apps/prairielearn/assets/scripts/instructorAssessmentManualGradingRubricEditingPreact.tsx

```
@@ -0,0 +1,28 @@
+// import assert from 'node:assert';
+
+import { onDocumentReady } from '@prairielearn/browser-utils';
+import { render } from '@prairielearn/preact-cjs';
+
+import { AssessmentQuestionRubricTable } from '../../src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestionRubricTablePreact.html.js';
+
+onDocumentReady(() => {
+  const rubricTable = document.querySelector('#rubric-editing-preact') as HTMLElement;
+  // assert(rubricTable.dataset.csrfToken);
+  // assert(rubricTable.dataset.assessmentQuestion);
+  if (!rubricTable.dataset.csrfToken || !rubricTable.dataset.assessmentQuestion) {
+    return;
+  }
```

I'm using this in place of an assertion. For some reason I can't get the import to work for assert.

---

This is a client side script, it can't assert. You should throw an error instead.